library/ajax/login_counter_ip_tracker.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * login_counter_ip_tracker.php
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2023 Brady Miller <brady.g.miller@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Auth\AuthUtils;
+use OpenEMR\Common\Csrf\CsrfUtils;
+
+require_once(__DIR__ . "/../../interface/globals.php");
+
+if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"], 'counter')) {
+    CsrfUtils::csrfNotVerified(false);
+}
+
+if (!AclMain::aclCheckCore('admin', 'super')) {
```

Comment:
Do you think frontend staff are going to need the ability to reset someone from being logged out?  This seems like a fairly common support staff function.  I understand the desire to lock it down to just an admin thing... but it seems like the support staff are going to hit this the most from someone.

Reply:
changed this to admin users acl (same that is needed to get to the admin users gui)